### DISCUSSION TITLE
[WI] Wisconsin counties red and yellow

### DISCUSSION THREAD

**Discussion by kmmurray108** [kuriwaki/cvr_harvard-mit_scripts#307] [General]
### All Counties

Undervotes were reported in the Harvard dataset but not in MEDSL or precinct dataset
Howie Hawkins (Green party candidate for president) is missing from Harvard and MEDSL datasets
Precinct dataset reports vote totals for several write-in candidates while Harvard and MEDSL datasets do not

### Yellow

**Brown** (#68)
Vote total mismatches:
Harvard and MEDSL datasets match for Presidential candidates, slight discrepancy from the precinct data. Precinct dataset values are validated by the county [website](https://www.browncountywi.gov/i/f/files/County-Clerk/Elections/Election%20Results/2020/November/Official%20Summary.pdf)

Harvard and MEDSL datasets match for US House Dist. 8 candidates, slight discrepancy from the precinct data. Precinct dataset values are validated by the county [website](https://www.browncountywi.gov/i/f/files/County-Clerk/Elections/Election%20Results/2020/November/Official%20Summary.pdf)

Harvard and MEDSL datasets match for State Senate Dist. 2 candidates, slight discrepancy from the precinct data. Precinct dataset values are validated by the county [website](https://www.browncountywi.gov/i/f/files/County-Clerk/Elections/Election%20Results/2020/November/Official%20Summary.pdf)

Harvard and MEDSL datasets match for Eric Wimberger (State Senate Dist. 30), slight discrepancy from the precinct data. Precinct dataset values are validated by the county [website](https://www.browncountywi.gov/i/f/files/County-Clerk/Elections/Election%20Results/2020/November/Official%20Summary.pdf)

Harvard and MEDSL datasets match for Kathy Hinkfuss (State House Dist. 4), slight discrepancy from the precinct data. Precinct dataset values are validated by the county [website](https://www.browncountywi.gov/i/f/files/County-Clerk/Elections/Election%20Results/2020/November/Official%20Summary.pdf)

Harvard and MEDSL datasets match for State House Dist. 6 candidates, slight discrepancy from the precinct data. Precinct dataset values are validated by the county [website](https://www.browncountywi.gov/i/f/files/County-Clerk/Elections/Election%20Results/2020/November/Official%20Summary.pdf)

Harvard and MEDSL datasets match for John Macco (State House Dist. 88), slight discrepancy from the precinct data. The county validates precinct dataset values [website](https://www.browncountywi.gov/i/f/files/County-Clerk/Elections/Election%20Results/2020/November/Official%20Summary.pdf)

**Dane** (#76)
Harvard and MEDSL vote totals match for all candidates but differ from the precinct dataset vote totals in all cases except for Joan A Ballweg (State Senate Dist. 14), State House Dist. 37 candidates, John Plumer (State House Dist. 42), Phil Anderson (State House Dist. 47), and David J Dahlke (State House Dist. 81). Precinct dataset values are validated by the county [website](https://elections.countyofdane.com/Election-Result/124). 

**Sheboygan** (#92)
Harvard and MEDSL vote totals match for all candidates but differ from the precinct dataset vote totals in all cases. Precinct dataset values match the certified totals as reported by the canvass reporting [system](https://www.presidency.ucsb.edu/sites/default/files/election_data/wi-2020.pdf).

**Waukesha**  (#95)
Harvard and MEDSL vote totals match for all candidates but differ from the precinct dataset vote totals in all cases except for Presidential candidate Jo Jorgensen, Bryan Steil (US House Dist. 4), State Senate Dist. 8 candidates, Julian Bradley (State Senate Dist. 28), State House Dist. 14 candidates, State House Dist. 15 candidates, State House Dist. 22 candidates, State House Dist. 24 candidates, Cory Horlacher (State House Dist. 33), State House Dist. 38 candidates, Chuck Wichgers (State House Dist. 83), State House Dist. 84 candidates, and Aaron D Perry (State House Dist. 97).

Precinct dataset values are validated by the county [website](https://www.waukeshacounty.gov/globalassets/county-clerk/election-information/election-results/2020/state-boc-signed-certifcation2020.pdf).

### Red

**Calumet** (#75)

Over 50% of the votes is absent

**Eau Claire**
US House Dist. 3 candidates in the MEDSL dataset are misaligned with the others 

Harvard and MEDSL vote totals match for all candidates but differ from the precinct dataset vote totals in all cases except for Charlene Charlie Warner (State House Dist. 93). Precinct dataset values are validated by the county [website](https://www.eauclairecounty.gov/home/showdocument?id=38118). 

**Jefferson**
US House Dist. 5 candidates in the MEDSL dataset are misaligned with the others 

State House Dist. 33, 37, and 38 candidates missing from Harvard and MEDSL datasets
Harvard and MEDSL vote totals match for all candidates but differ from the precinct dataset vote totals in all cases. Precinct dataset values are validated by the county [website](https://apps.jeffersoncountywi.gov/jc/election/results/11032020). 

**Kenosha** (#83)

Mislabeled USHOUSE candidate

**Outagamie**
Jessica J King and Glenn Grothman (US House Dist. 6 candidates) are not included in the official verified vote total on the county [website](https://www.outagamie.org/home/showpublisheddocument/78479/637406151338730000) (and therefore indicates they were not on the ballot in Outagamie county). These candidates are in the Harvard and MEDSL datasets but not the precinct dataset.

Harvard and MEDSL datasets are missing State Senate Dist. 14 candidates and State House Dist. 2 candidates.

Harvard and MEDSL vote totals match for all candidates but differ from the precinct dataset vote totals in all cases except for Jim Steineke (State House Dist. 5), Richard Sarnwick (State House Dist. 6), and State House Dist. 55 candidates. Precinct dataset values are validated by the county [website](https://www.outagamie.org/home/showpublisheddocument/78479/637406151338730000). 

**Pierce**
Tricia Zunker and Tom Tiffany (US House Dist. 7 candidates) are not included in the official verified vote total on the county [website](https://cms5.revize.com/revize/piercewi/Document_center/Community/Department/Election/11-03-2020%20General%20Election.pdf) (and therefore indicates they were not on the ballot in Pierce county). These candidates are in the Harvard and MEDSL datasets but not the precinct dataset.

Harvard and MEDSL vote totals match for all candidates but differ from the precinct dataset vote totals in all cases. Precinct dataset values are validated by the county [website](https://cms5.revize.com/revize/piercewi/Document_center/Community/Department/Election/11-03-2020%20General%20Election.pdf). 

**Sauk**
US House Dist. 2 candidates in the MEDSL dataset are misaligned with the others

Harvard and MEDSL vote totals match for all candidates but differ from the precinct dataset vote totals in all cases except for State House Dist. 51. Precinct dataset values match the certified totals as reported by the canvass reporting [system](https://www.presidency.ucsb.edu/sites/default/files/election_data/wi-2020.pdf).

**St. Croix**
Harvard and MEDSL datasets are missing State House Dist. 93 candidates

Harvard and MEDSL vote totals match for all candidates but differ from the precinct dataset vote totals in all cases except for Kim Butler (State House Dist. 28), and State House Dist. 75 candidates. Precinct dataset values match the certified totals as reported by the canvass reporting [system](https://www.presidency.ucsb.edu/sites/default/files/election_data/wi-2020.pdf).

**Wood**
MEDSL data is missing entirely

Harvard vote totals differ from the precinct dataset vote totals in all cases.  Precinct dataset values are validated by the county [website](https://www.woodcountywi.gov/departments/clerk/election/Default.aspx?ElectionID=20GWIWOO).

### Takeaways
In some cases, vote totals are missing entirely. There are also several instances where candidate names are mismatched between datasets, which may account for discrepancies. Additionally, the fact that undervotes were reported in the Harvard dataset but not in MEDSL or precinct dataset; Howie Hawkins (Green party candidate for president) is missing from Harvard and MEDSL datasets; and the precinct dataset reports vote totals for several write-in candidates while Harvard and MEDSL datasets do not may also be causing issues. 

In every county, the precinct vote totals match the official certified tallies.

**Comment by kuriwaki** [comment]
Among 243 county x D/R candidate totals in Wisconsin, 

* 55 exactly match the precinct data 
* 108 where Harvard counts are less than the precinct data (differences as a percentage of the precinct count is has a 10-90 percentile of [-39%, -0.01%])
* 39 where Harvard counts are more than the precinct data (differences as a percentage of the precinct count is has a 10-90 percentile of [+0.09%, +34%])
* 32 are mis-matched rows of some sort

_all_ of the 13 counties here have at _least one_ such discrepancy (none are perfect)

**Comment by mreece13** [comment]
@kmmurray108 Can you break these into single issues, it would be a lot easier to understand and address them? E.g., every erroring candidate x county should be an independent issue.

**Comment by kmmurray108** [comment]
@mreece13 yes, for sure. I'll leave this issue open so we have them all together, but I'll go ahead and make separate issues for every error in each county.

**Comment by kuriwaki** [comment]
@kmmurray108 Dane, Brown, Waukesha, and Sheboygan are now classified as yellow, not red.  (the status as of today is [here](https://github.com/kuriwaki/cvr_harvard-mit_scripts/blob/a5d4fbc46445051cc1fccd9d90e3d2f82052fcd8/status/counties-classified/WISCONSIN.txt) -- we are starting to track this in the github). 


I am therefore renaming this mega-issue into yellow and red, and I updated your original post to reflect.
